src/server/database/knexfile.js

```diff
@@ -9,11 +10,22 @@ export default {
 
   connection: {
     host: process.env.POSTGRES_HOST,
-    database: process.env.POSTGRES_DATABASE,
+    database:
+      process.env.NODE_ENV === 'test'
+        ? 'zola-test'
```

Comment:
I guess it’s fine if it’s not a variable?

Reply:
Yes it's fine.